lib/core/src/main/java/io/atlasmap/core/AtlasUtil.java

```diff
@@ -104,7 +105,7 @@ protected static void validateUri(String atlasUri) {
             pass1 = atlasUri.split("\\?", 2);
         }
 
-        List<String> parts = new ArrayList<String>();
```

Comment:
here too.

Reply:
I left this one because there are several other changes to this same class.